src/server/api/auth.ts

```diff
@@ -84,6 +85,69 @@ authRouter.post("/register", async (req, res, next) => {
     }
 })
 
+// POST /api/auth/login
+authRouter.post("/login", async (req, res, next) => {
+  try {
+    const { email, password } = req.body;
+
+    const user = await prisma.user.findUniqueOrThrow({
+        where: {
+          email: email,
+        },
+      });
+  
+
+    // if (user) {
+    bcrypt.compare(
+      password,
+      user.password,
+      async function (err: Error | undefined, result: boolean) {
+        if (err) {
+          next(err)
+        } else if (result) {
+          // JSON Web Token returned to client
+          // TODO This solves an error, investigate other way to solve
+          const token = ACCESS_TOKEN_SECRET ?
+          jwt.sign({
+            username: user.username,
+            id: user.id,
+          },
+            ACCESS_TOKEN_SECRET) : 
+            next({
+              name: "InvalidAccessTokenSecret", 
+              message: "Access token secret is undefined"
```

Comment:
We can send a 401 here.

```suggestion
            res.status(401);
            next({
              name: "InvalidAccessTokenSecret", 
              message: "Access token secret is undefined"
```